mozci/task.py

```diff
@@ -451,6 +452,24 @@ def failing_tasks(self):
             )
         ]
 
+    @memoized_property
+    def is_config_consistent_failure(self) -> Optional[bool]:
+        config_to_results = collections.defaultdict(list)
+        for task in self.tasks:
+            for result in task.results:
+                if result.group == self.name:
+                    config_to_results[task.configuration].append(result.ok)
+
+        # If there is no config for which we have at least two runs, return None (that is, unknown).
+        if all(len(results) <= 1 for results in config_to_results.values()):
+            return None
+
+        # Return True if there is at least one configuration for which we have only failures, False otherwise.
+        return any(
+            len(results) >= 2 and all(not result for result in results)
```

Comment:
nit: I think the `all(not result for result in results)` could be simplified to `not any(results)`?